extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -14,23 +14,99 @@
  * limitations under the License.
  */
 
+.i-amphtml-story-interactive-slider-container {
+  --percentage: 0%;
+}
+
 .i-amphtml-story-interactive-slider-input-container {
+  display: flex !important;
   background: var(--i-amphtml-interactive-option-background-color) !important;
   border-radius: 2em !important;
   box-shadow: var(--i-amphtml-interactive-component-shadow) !important;
+  align-items: center !important;
+  justify-content: center !important;
+  height: 100% !important;
+  width: 100% !important;
+  padding: 0em 1em !important;
+  box-sizing: border-box !important;
+  position: relative !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble {
+  position: absolute !important;
+  transform: translate(-50%, -50%) scale(0) !important;
+  width: 2.8em !important;
+  height: 2.8em !important;
+  top: 50% !important;
+  left: var(--percentage) !important;
+  text-align: center !important;
+  line-height: 2.9em !important;
+  z-index: 2 !important;
+  font-weight: 500 !important;
+  color: white !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble.show {
+  transform: translate(-50%, -155%) scale(1) !important;
+}
+
+.i-amphtml-story-interactive-slider-bubble::after {
+  position: absolute !important;
+  content: '' !important;
+  box-sizing: border-box !important;
+  height: 100% !important;
+  width: 100% !important;
+  background: var(--interactive-accent-color) !important;
+  z-index: -1 !important;
+  left: 50% !important;
+  transform: translateX(-50%) rotate(45deg) !important;
+  border-radius: 50% 50% 0% 50% !important;
+  box-shadow: 0em 0em 0.5em rgba(0, 0, 0, 0.25) !important;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-bubble {
+  font-weight: 700 !important;
+  transform: translate(-50%, -50%) scale(0.8) !important;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-slider-bubble::after {
+  cursor: initial !important;
+  border: 0.125em solid white !important;
+  border-bottom-right-radius: 50% !important;
+}
+
+.i-amphtml-story-interactive-slider-input-container .i-amphtml-story-interactive-side-values {
+  font-size: 1.125em !important;
+  color: var(--interactive-accent-color) !important;
+  font-weight: 600 !important;
```

Comment:
The left and right numbers are an optional display type and should be removed for this first PR.
See comments below.